apps/hash-frontend/src/pages/login.page.tsx

```diff
@@ -15,14 +15,35 @@ import {
 } from "react";
 
 import { useHashInstance } from "../components/hooks/use-hash-instance";
-import { useLogoutFlow } from "../components/hooks/use-logout-flow";
+import { ArrowRightToBracketRegularIcon } from "../shared/icons/arrow-right-to-bracket-regular-icon";
+import { ArrowTurnDownLeftRegularIcon } from "../shared/icons/arrow-turn-down-left-regular-icon";
 import { getPlainLayout, NextPageWithLayout } from "../shared/layout";
-import { Button } from "../shared/ui";
+import { Button, ButtonProps } from "../shared/ui";
+import { AuthHeading } from "./shared/auth-heading";
 import { useAuthInfo } from "./shared/auth-info-context";
+import { AuthLayout } from "./shared/auth-layout";
+import { AuthPaper } from "./shared/auth-paper";
 import { mustGetCsrfTokenFromFlow, oryKratosClient } from "./shared/ory-kratos";
 import { useKratosErrorHandler } from "./shared/use-kratos-flow-error-handler";
 import { WorkspaceContext } from "./shared/workspace-context";
 
+const SignupButton = styled((props: ButtonProps) => (
+  <Button variant="secondary" size="small" {...props} />
+))(({ theme }) => ({
+  color: theme.palette.common.white,
+  background: "#1F2933",
+  transition: theme.transitions.create(["background", "box-shadow"]),
+  borderColor: "#283644",
+  boxShadow: theme.shadows[3],
+  "&:hover": {
+    background: "#283644",
+    boxShadow: theme.shadows[4],
+    "&:before": {
+      opacity: 0,
+    },
+  },
+}));
+
 const LoginPage: NextPageWithLayout = () => {
```

Comment:
I appreciate most of the app isn't responsive, but can we make this page mobile-friendly since we're touching it?

Hopefully just be a case of reducing the padding on the form and wrapping the signup button down

Reply:
Have made it responsive in https://github.com/hashintel/hash/pull/4009/commits/e9ec95543db9dbf2b331222b8d40941b1d8e7f83, now looks like this:

https://github.com/hashintel/hash/assets/42802102/837cbdc1-e0d4-49df-8377-1713537b3ade